UI/ChatWnd.cpp

```diff
@@ -195,13 +195,13 @@ void MessageWndEdit::FindGameWords() {
             m_game_words.insert(UserString(entry.second->Name()));
     }
     // add ship hulls
-    for (const auto& entry : GetPredefinedShipDesignManager()) {
-        if (entry.second->Hull() != "")
-            m_game_words.insert(UserString(entry.second->Hull()));
+    for (const auto& design : GetPredefinedShipDesignManager().GetOrderedShipDesigns()) {
```

Comment:
Even if there is a reason to add only seen hulls, it's still not enough to handicap the feature when players can see not-yet-available hulls in the pedia and might want to refer to them by name.